### ISSUE TITLE
new twitch feature "playlists beta" doesn't load up in livestreamer

### ISSUE THREAD

**Issue by iLLsen** [chrippa/livestreamer#1087]
The new Playlists feature isnt working.
For example right now http://www.twitch.tv/swiftor is streaming in this new mode.

Livestreamer returns "error: No streams found on this URL: http://www.twitch.tv/swiftor"
same with http://www.twitch.tv/futuremangaming

**Comment by GTB3NW** [comment]
Yeah I have issues watching Lirik, to be fair I have issues even watching in the browser right now :S

**Comment by PureTryOut** [comment]
This issue is still here. Would love to see this fixed sometime.

**Comment by Songworks** [comment]
On a related sidenote: A twitch channel is also considered as offline if it is hosting another channel.

**Comment by Songworks** [comment]
Playlists appear to be working with livestreamer nowdays.
